sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java

```diff
@@ -129,12 +138,90 @@
      *
      * @param countryCode The ISO 3166-2 country code.
      * @param searchRequest {@link PhoneNumberSearchRequest} specifying the search request
+     * @param pollInterval The time our long running operation will keep on polling
+     * until it gets a result from the server
      * @return A {@link PollerFlux} object with the reservation result
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<PollResult<PhoneNumberSearchResult>, PhoneNumberSearchResult> beginSearchAvailablePhoneNumbers(
-        String countryCode, PhoneNumberSearchRequest searchRequest) {
-        return null;
+    public PollerFlux<PhoneNumberOperationResult, PhoneNumberSearchResult> beginSearchAvailablePhoneNumbers(
+        String countryCode, PhoneNumberSearchRequest searchRequest, Duration pollInterval) {
```

Comment:
`pollInterval` is now configurable on `PollerFlux` type itself. So, client APIs don't have to include this param.